Build/src/V1/Gapic/CloudBuildGapicClient.php

```diff
@@ -388,14 +388,16 @@ public static function buildName($project, $build)
      * build_trigger resource.
      *
      * @param string $project
+     * @param string $location
      * @param string $trigger
      *
      * @return string The formatted build_trigger resource.
      */
-    public static function buildTriggerName($project, $trigger)
+    public static function buildTriggerName($project, $location, $trigger)
```

Comment:
This is a breaking change due to cl/547283196 and should be fixed to safely merge unless its a conscious decision by the product.